docs/src/tutorial.md

```diff
@@ -5,19 +5,33 @@ NLPModelsKnitro is a thin KNITRO wrapper for NLPModels. In this tutorial we show
 Pages = ["tutorial.md"]
 ```
 
-## Simple problems
-
 Calling KNITRO is simple:
-```@docs
-knitro
+```julia
+`output = knitro(nlp; kwargs...)`
+
+Solves the `NLPModel` problem `nlp` using KNITRO.
+
+# Optional keyword arguments
+* `x0`: a vector of size `nlp.meta.nvar` to specify an initial primal guess
+* `y0`: a vector of size `nlp.meta.ncon` to specify an initial dual guess for the general constraints
+* `z0`: a vector of size `nlp.meta.nvar` to specify initial multipliers for the bound constraints
+* `callback`: a user-defined `Function` called by KNITRO at each iteration.
+
+For more information on callbacks, see [https://www.artelys.com/docs/knitro/2_userGuide/callbacks.html](https://www.artelys.com/docs/knitro/2_userGuide/callbacks.html) and
+the docstring of `KNITRO.KN_set_newpt_callback`.
+
+All other keyword arguments will be passed to KNITRO as an option.
+See [https://www.artelys.com/docs/knitro/3_referenceManual/userOptions.html](https://www.artelys.com/docs/knitro/3_referenceManual/userOptions.html) for the list of options accepted.
```

Comment:
Is there a way to import the actual docstring here?

Reply:
Unfortunatly no. Whenever we try a block like
```@docs
knitro
```
this will try to access the module, which cannot be loaded...